scene/main/node.cpp

```diff
@@ -1442,7 +1442,19 @@ Node *Node::get_node_or_null(const NodePath &p_path) const {
 
 Node *Node::get_node(const NodePath &p_path) const {
 	Node *node = get_node_or_null(p_path);
-	ERR_FAIL_COND_V_MSG(!node, nullptr, "Node not found: " + p_path + ".");
+
+	if (p_path.is_absolute()) {
+		ERR_FAIL_COND_V_MSG(
+				!node,
+				nullptr,
+				vformat(R"(Node not found: "%s" (absolute path attempted from "%s").)", p_path, get_path()));
+	} else {
+		ERR_FAIL_COND_V_MSG(
+				!node,
+				nullptr,
+				vformat(R"(Node not found: "%s" (relative to "%s").)", p_path, get_path()));
+	}
```

Comment:
I'm still not fond of this kind of "one arg per line" approach which takes a lot of vertical space for something which shouldn't take a lot of developer attention (it's an assert, move along).

IMO this is much better:
```
	if (p_path.is_absolute()) {
		ERR_FAIL_COND_V_MSG(!node, nullptr,
				vformat(R"(Node not found: "%s" (absolute path attempted from "%s").)", p_path, get_path()));
	} else {
		ERR_FAIL_COND_V_MSG(!node, nullptr,
				vformat(R"(Node not found: "%s" (relative to "%s").)", p_path, get_path()));
	}
```